stratum/hal/lib/barefoot/bf_chassis_manager.cc

```diff
@@ -550,6 +575,12 @@ ::util::Status BFChassisManager::ReplayPortsConfig(uint64 node_id) {
           unit, port_id, *config.autoneg));
       config_new->autoneg = *config.autoneg;
     }
+    if (config.loopback_mode &&
+        config.loopback_mode != LOOPBACK_STATE_UNKNOWN) {
```

Comment:
Isn't this testing the same condition? `UNKNOWN = 0`

Reply:
config.loopback_mode is an `absl::optional` type
the bool operator of `absl::optional` returns true if the value is present
and `!=` operator will compare the actual value in the `absl::optional` variable